internal/server/grpc.go

```diff
@@ -45,9 +45,21 @@ type RunOpts struct {
 	ClusterDomain string
 	IPCidr        string
 	KubeContext   string
+
+	// SkipNamespaces is a list of namespaces to skip when forwarding
+	// services. Defaults to "kube-system" when passed to NewGRPCService.
+	SkipNamespaces []string
 }
 
 func NewGRPCService(opts *RunOpts) *GRPCService {
+	if opts == nil {
+		opts = &RunOpts{}
+	}
+
+	if opts.SkipNamespaces == nil {
+		opts.SkipNamespaces = []string{"kube-system"}
+	}
```

Comment:
By default skip port-forwarding in the kube-system namespace.

Reply:
Q: Is there ever any situation where port-forwarding to kube-system would actually be desired? I'm picturing a scenario where someone assumes the default behavior and passes in a namespace <foo> to skip, and then <foo> is skipped but now `kube-system` is no longer skipped.

Reply:
Yeah, that's fair. We might just want to always add `kube-system`.